src/test/java/uk/gov/pay/adminusers/unit/service/ServiceResourceUpdateTest.java

```diff
@@ -143,39 +120,12 @@ public void shouldUpdateName_andServiceNameEn_whenReplaceServiceNameEn() {
     }
 
     @Test
-    public void shouldUpdateName_ServiceNameEn_andServiceNameCy_whenReplaceName_AndReplaceServiceNameCy_withNoExistingServiceNameCy() {
+    public void shouldUpdateName_ServiceNameEn_andServiceNameCy_whenReplaceServiceNameEn_AndReplaceServiceNameCy_withNoExistingServiceNameCy() {
```

Comment:
this really does my head in. how about `should_update_existing_en_service_name_and_non_existing_cy_service_name`?

Reply:
I’ll have a look at the names for all the tests in this class to keep them consistent.